### ISSUE TITLE
Image external link as email address?

### ISSUE THREAD

**Issue by buarrows** [silexlabs/Silex#464]
Is it possible to add an external link to an image that is a mailto address (i.e. email address)?

**Comment by singchan** [comment]
Hmm, you can put "mailto:" in the link property, but you are looking for a way to protect your email address?

**Comment by buarrows** [comment]
No – I don’t need to protect it. I thought of using the mailto:

Also, silex seems to be crashing a lot. Is this a common problem? I am using it on Safari on a Macbook with the latest iOS.

**Comment by singchan** [comment]
Yes you can use mailto: on an image I believe

Safari is not officialy supported, only firefox and chrome. When you say crash what do you mean exactly?

**Comment by buarrows** [comment]
When I am in Safari, if I am working on the page, it will just randomly crash. I will see about using chrome or firefox to see if I have the same problem.

Thanks,
Joyce

**Comment by singchan** [comment]
Ok son safari closes? Or do you have a "this script is slow, wait or kill" ?

**Comment by buarrows** [comment]
It actually just closes and I have to reopen the file. I don’t get an error message.

**Comment by buarrows** [comment]
One other question. I made a newsletter webpage using silex, and for the mobile view, it shows a menu box (box with a few lines in it). I only have one page so I don’t need that box – is there a way to remove that box? Also, I assume that the page that I work on is Page 1 by default – is that right?

Thanks,
joyce

**Comment by singchan** [comment]
Yes this is right
You can select the menu in silex and uncheck "visible on mobile"

**Comment by buarrows** [comment]
Great – thanks so much!

**Comment by buarrows** [comment]
Another question. I was unable to change the publish settings and log onto my server through Silex, but I was able to just drag the file to the server using FileZilla (on PC) to publish the page. Is there something I have to coordinate with my university IT?

Best,
Joyce

**Comment by singchan** [comment]
Are you able to connect from outside your university?

You can use filezilla but transfer the published website as it is optimized and cleaned up

**Comment by buarrows** [comment]
Yes, I am able to connect normally from outside my university. The information is here for my university:
https://www.bu.edu/tech/services/comm/websites/www/non-wordpress/how-to/upload/

I get this error message when I try to ftp (this time I am using chrome): it is on editor.silex.me/api/v1.0/ftp-auth-submit

I put in my server as people.bu.edu and the port 22 and then put in my username and password then get this error:
Proxy Error
The proxy server received an invalid response from an upstream server.
The proxy server could not handle the request POST /api/v1.0/ftp-auth-submit<http://editor.silex.me/api/v1.0/ftp-auth-submit>.
Reason: Error reading from remote server

**Comment by singchan** [comment]
Ok then you probably need to publish in silex and transfer by hand with filezilla...

Please let me know if you still have crashes

**Comment by buarrows** [comment]
I haven’t had crashes since I have been using Google Chrome for the editor in silex.
I have another question though – is it possible to add code to have a link in a text box open a new tab or window when someone clicks on the link? Right now it doesn’t seem to be an option and probably requires addition of some code?

Thanks!

**Comment by singchan** [comment]
> I haven’t had crashes since I have been using Google Chrome for the editor in silex. 

ok good to know

> I have another question though – is it possible to add code to have a link in a text box open a new tab or window when someone clicks on the link? Right now it doesn’t seem to be an option and probably requires addition of some code?

yes that is right, the option is missing.. only when you set a link on an element on stage with the "link to" box it opens in _blank

you could add this script to your JS scripts
```js
$('a[href]').attr('target', '_blank'); 
```
It will make all links open in a new tab or window. I did not test it, but it is simple, it should work

do not hesitate to ask questions about the script if you are interested in JS

**Comment by buarrows** [comment]
Thanks so much – this worked!

**Comment by buarrows** [comment]
Is there someone in your group who can help with tweaking html5 code for compatibility with emailing out a webpage made from silex (I assume in html5) in programs like outlook?

**Comment by lexoyo** [comment]
you mean send an email with an HTML content made with Silex?
please open a new issue, this is an interesting subject

i believe these libs can do the job

* https://github.com/Automattic/juice
* https://www.npmjs.com/package/inline-css

**Comment by buarrows** [comment]
Yes I will open a new issue.

**Comment by buarrows** [comment]
I’m sorry for asking so many questions, but I was able to download npm on windows and can switch the path to where the html file is.

I am not sure how to do the inline-css to make it readable in the outlook email file. I chose silex because it didn’t require much coding, but I am willing to learn what is needed.

Would it be possible for me to send you the .html file for you to see if it is possible for it to be altered with the below libraries so it can then be placed into an outlook ‘insert as text’ email that has correct formatting?

Joyce
